server.go

```diff
@@ -95,6 +96,9 @@ Loop:
 				if err != nil {
 					echoerr(c, "listen: drop: client id should be a number")
 				} else {
+					if c2, ok := gConnList[id]; ok {
+						c2.Close()
```

Comment:
Before you added this, did the client close the connection? Is it OK if first the server closes the connection and then the client closes it a second time?